src/main/java/fr/insee/pearljam/infrastructure/security/config/OidcSecurityConfiguration.java

```diff
@@ -186,6 +186,8 @@ private void authorizeRequests(HttpSecurity http) throws Exception {
 						.hasAnyRole(adminRole, localUserRole, nationalUserRole)
 						.requestMatchers(HttpMethod.GET, Constants.API_CAMPAIGNS_ID_ON_GOING)
 						.hasRole(adminRole)
+						.requestMatchers(HttpMethod.GET, Constants.API_CAMPAIGNS_ON_GOING)
```

Comment:
suggestion: add WEBCLIENT role in allowed roles for this endpoint (see line 93 for example)